appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/WalletUtils.java

```diff
@@ -93,25 +111,24 @@ public static void createInstallWalletNotification() {
       notificationManager.notify(0,
           buildNotification(Integer.toString(POA_NOTIFICATION_ID), intent));
     } else {
-      Notification notificationHeadsUp =
-          buildNotificationOlderVersion(intent);
+      Notification notificationHeadsUp = buildNotificationOlderVersion(intent);
       notificationManager.notify(POA_NOTIFICATION_ID, notificationHeadsUp);
     }
   }
 
   private static Intent getNotificationIntent() {
     String url = URL_INTENT_INSTALL;
-    boolean hasAptoide = hasAptoideInstalled();
 
-    if (hasAptoide) {
+
+    if (WalletUtils.getAptoideVersion() != UNINSTALLED_APTOIDE_VERSIONCODE) {
```

Comment:
Do we need to get Aptoide version to check if it is installed? If so then the variable used as comparison (UNINSTALLED_APTOIDE_VERSIONCODE) then it should be set as the default value in the getAptoideVersion method.